third_party/bazel_rules/rules_kotlin/kotlin/native/wasm.bzl

```diff
@@ -0,0 +1,3 @@
+# Alias wasm split transition rule unless G3 Kotlin Native synced with GH rules
+def wasm_kt_binary(name, kt_target):
+    native.alias(name = name, actual = kt_target, visibility = ["//visibility:public"])
```

Comment:
What is `native`? Can you point me to some documentation for me edification?

Reply:
It means "built in bazel function", otherwise you need a load() statement to load the implementation of alias.